sass/components/_emoticons.scss

```diff
@@ -23,6 +23,7 @@
     min-width: 21px;
     vertical-align: middle;
     width: 21px;
+    color: transparent;
```

Comment:
I'm simply making the text "invisible" using styling because both `visibility: hidden` and `display: none` result in a non-copyable element.